halconfig/settings.js

```diff
@@ -177,6 +177,7 @@ window.spinnakerSettings = {
     pipelines: true,
     pipelineTemplates: pipelineTemplatesEnabled,
     roscoMode: true,
+    slack: false,
```

Comment:
This would be useful for halyard users, I'm sure.  We can certainly address this in a future PR.

Reply:
great! I wanted to disable in OSS to start since you also need to have a token.